CHANGELOG.unreleased.md

```diff
@@ -2,6 +2,8 @@
 
 ### Enhancements
 
+- [VDI migration] Display shared and local SRs first in the selector [#3945](https://github.com/vatesfr/xen-orchestra/issues/3945) (PR [#3996](https://github.com/vatesfr/xen-orchestra/pull/3996))
```

Comment:
I don't know what it means: "shared and local SRs" = "all the SRs".

Reply:
shared and local SRs from the same pool of VM.

Reply:
Then "display same-pool SRs first in the selector".

Reply:
Display same-pool SRs first in the selector ?

Reply:
Timing :tada: